plugins/scripts/Optimobile.plist

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should more mimic the credentials file of https://github.com/Kumulos/KumulosSdkCordova/blob/master/scripts/kumulos.plist

Reply:
(can be handled in the second PR)